### DISCUSSION TITLE
[Feature] Navigate to photos from the same day or location from the info panel

### DISCUSSION THREAD

**Discussion by dgtal1** [immich-app/immich#9076] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When browsing photos would be great to be able to quickly navigate to other photos from the same date or location. I envision this as hyperlinks attached to the information in the side info panel (marked yellow):
![image](https://github.com/immich-app/immich/assets/27864579/ac0c9ab2-e24d-4411-8e7b-88fe941868e9)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4115
